pkg/brokerapi/fake/server/handler.go

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"context"
+
+	"github.com/pivotal-cf/brokerapi"
+)
+
+// Handler is a fake implementation oif a brokerapi.ServiceBroker
+type Handler struct {
+	Catalog         []brokerapi.Service
+	CatalogRequests int
```

Comment:
Can you add a comment to explain why this is just an "int" while the other ones are slices of structs?
Just something like:
```
// Since there is no incoming data to save (unlike the other APIs) we just keep a count
// of the requests.
```